scania-digital-design-system/tegel
# feat(block): prefix change, sdds -> tds

**Describe pull-request**  
Prefix change in block component

**Solving issue**  
Fixes: DTS-1626

**How to test**  
1. Go to the link below
2. Check Block component
3. Make sure the component works and looks as in the main version of Storybook
